.README/README-KO.md

```diff
@@ -0,0 +1,48 @@
+<p align="center">
+  <img
+    src="https://raw.githubusercontent.com/rust-analyzer/rust-analyzer/master/assets/logo-wide.svg"
+    alt="rust-analyzer logo">
+</p>
+
+<p align="center">
+  <a href="https://github.com/rust-lang/rust-analyzer/blob/master/README.md">en</a>
+  <a href="#">ko</a>
+</p>
+
+러스트 분석기는 러스트 언어를 위한 모듈형 컴파일러 프론트엔드입니다.
+이는 Rust에 대한 우수한 IDE 지원을 만들기 위한 rls-2.0 노력의 일환입니다.
+
+## 빠른 시작
+
+https://rust-analyzer.github.io/manual.html#installation
+
+## 설명
+
+Rust에 **기여하는 방법**이 궁금하다면 [.docs/dev](.docs/dev)폴더 확인하세요.
```

Comment:
Wrong link here. It's the same in the original README, but this one points to https://github.com/rust-lang/rust-analyzer/blob/master/.README/.docs/dev. I think it's fine to use a full URL or whatever.